### PR TITLE
Improvement on login, fix group IDs

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Improve login method to avoid waiting for the whole page to load, the accessible menu should be enough
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | ~
| Sponsor company   | ~
| How to test?      | See https://github.com/PrestaShop/PrestaShop/pull/37379

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
